eth_pydantic_types/hashing.py

```diff
@@ -0,0 +1,13 @@
+from typing import Tuple
+
+
+def get_hash_pattern(str_size: int) -> str:
```

Comment:
Couple things here!

1. You can use a constant-format string. Strings in python have a method called `.format`.
```
HASH_PATTERN = "^0x[a-fA-F0-9]{{{str_size}}}$"

# Then when you go to use it

hash_pattern = HASH_PATTERN.format(str_size)
# and it will automatically put the `str_size` in the {str_size} part`

```

2. Use `re.compile` and the Pattern type for performance and better type-checking
```python
import re

HASH_PATTERN = re.compile("..your pattern ... ")
```

3. I think in python regex, you don't need the `$` or the `^` depending on what you are doing
```
HASH_PATTERN = re.compile(f"0x[a-fA-F0-9]{{{str_size}}}".format(str_size))
```